src/main/java/jenkins/plugins/logstash/LogstashConfiguration.java

```diff
@@ -27,6 +27,7 @@
 import jenkins.plugins.logstash.configuration.RabbitMq;
 import jenkins.plugins.logstash.configuration.Redis;
 import jenkins.plugins.logstash.configuration.Syslog;
+import jenkins.plugins.logstash.configuration.Logzio;
```

Comment:
more unused imports?

Reply:
For some reason, my changes were not committed. sorry.